src/Driver/PDO/Statement.php

```diff
@@ -145,8 +145,10 @@ public function execute($params = null): ResultInterface
      * @param int $type Parameter type
      *
      * @throws ExceptionInterface
+     *
+     * @return PDO::PARAM_*
      */
-    private function convertParamType(int $type): int
+    public static function convertParamType(int $type): int
```

Comment:
No, this method must not be part of DBAL's public API. Please make it private again.